src/device-registry/controllers/kafka/raw-device-measurements-consumer.js

```diff
@@ -1,52 +1,32 @@
-const kafka = require('kafka-node');
-const Consumer = kafka.Consumer;
 const { logObject, logElement } = require("../../utils/log");
```

Comment:
Could we try to avoid having files in many subfolders?
Having a near flat code organisational structure is good for many things, including the ease of maintenance + unit testing components. @NoahNsimbe

Reply:
thanks @Baalmart I am going to make the change....The motivation was to create a clear boundary between REST API logic and Message Broker logic because both provide exist and entry points for the microservice in totally different ways. 
 **1.** This could make it easy to identify and test message broker logic as well as the rest api logic. 
**2.** Making changes to the message broker shouldn't involve making changes to the rest api which makes maintenance, code update and review easy since a developer knows the folder to mess around with. In the future, if the microservice happens to have an embedded web interface for admin purposes then creating folders could be of great importance when the question "what do you want to change/update ??...is it the admin web interface, api endpoints or message broker connections??" is asked

Reply:
Thanks, all makes sense @NoahNsimbe ....was just worried about deeply nested files/directories. They usually becomes a maintenance nightmare since one has to navigate many subdirectories to reach their destination. You can read more about how to avoid deeply nested directory structures especially in our java code: https://stackoverflow.com/questions/26131352/avoid-deep-directory-structure-in-a-java-project